modules/restore-add-master.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/replacing-failed-master.adoc
+
+[id="restore-add-master_{context}"]
+= Adding a master host back to the etcd cluster
+
+Follow these steps to add a master host back to the etcd cluster. This procedure assumes that you previously removed the master host from the cluster and that its TLS certificates are valid.
```

Comment:
@retroflexer - @hexfusion commented in your draft about other etcd dependencies (such as valid DNS entries). 

Should we call out both of these? Or are there others where we should say something more generic about the etcd dependencies and use these two as examples?

Reply:
Yeah. etcd dependencies and use the two as examples.